sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -34,6 +34,8 @@ pi_result map_error(CUresult result) {
     return PI_INVALID_OPERATION;
   case CUDA_ERROR_INVALID_CONTEXT:
     return PI_INVALID_CONTEXT;
+  case CUDA_ERROR_NO_DEVICE:
```

Comment:
This error code seems to only come up with CUDA-OpenGL interop according to the docs - is that something you see otherwise in the wild?

Reply:
Sorry - this is actually a leftover from the first attempt I made, that was returning `PI_INVALID_PLATFORM` if there are no CUDA devices.

It should be fine to drop these two lines.

Reply:
Wondering if `PI_INVALID_PLATFORM` is the best match for the device error as I am unsure which OpenCL (extension) call would relate to OpenGL in this case?